ajax/libs/sticky-kit/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "sticky-kit",
+  "version": "1.1.3",
+  "description": "A jQuery plugin for creating smart sticky elements",
+  "filename": "sticky-kit.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/leafo/sticky-kit.git"
+  },
+  "author": {
+    "name": "leafo",
+    "email": "leafot@gmail.com"
+  },
+  "license": "MIT",
```

Comment:
The license needs a confirmation. On npm It show it's MIT in [bower.json](https://unpkg.com/sticky-kit@1.1.3/bower.json) but also shows WTFPL in [package.json](https://unpkg.com/sticky-kit@1.1.3/package.json). On GitHub, it shows it's MIT in package.json. On the website, it's also MIT.